prowler/providers/azure/services/monitor/monitor_ensure_diagnostic_setting_appropiate/monitor_ensure_diagnostic_setting_appropiate.py

```diff
@@ -0,0 +1,32 @@
+from prowler.lib.check.models import Check, Check_Report_Azure
+from prowler.providers.azure.services.monitor.monitor_client import monitor_client
+
+
+class monitor_ensure_diagnostic_setting_appropiate(Check):
+    def execute(self) -> Check_Report_Azure:
+        findings = []
+
+        for (
+            subscription_name,
+            diagnostic_settings,
+        ) in monitor_client.diagnostics_settings.items():
+            report = Check_Report_Azure(self.metadata())
+            report.status = "FAIL"
+            report.subscription = subscription_name
+            report.resource_name = "Monitor"
+            report.resource_id = "Monitor"
+            report.status_extended = f"There are no diagnostic settings capturing appropiate categories in subscription {subscription_name}."
+            for diagnostic_setting in diagnostic_settings:
+                if (
+                    diagnostic_setting.logs[0].enabled
+                    and diagnostic_setting.logs[1].enabled
+                    and diagnostic_setting.logs[3].enabled
+                    and diagnostic_setting.logs[5].enabled
```

Comment:
Why are you checking items 0, 1, 3 and 5 here? What is the reason behind?

Reply:
Im iterating through the logs and checking if the ones asked to be enabled ('Administrative','Security','Alert' and 'Policy') are enabled. Those numbers correspond to the order that those categories follow in the Azure Portal.
However, Im going to change it and upgrade it by checking also the category name so that way is guaranteed that even if more categories are set in the future, the code will still work.

Reply:
Perfect, thanks!